ai/extract_features.py

```
@@ -114,23 +112,26 @@ def batch_process(texts, batch_size=10000):
     return results
 
 
-def _extracted_from_extract_features_76(
-    vectorizer, df, category, n_components, random_state
-):
-    """Process smaller datasets all at once"""
-    # For smaller datasets, process all at once
+from typing import Any
```

Flake8 (7.2.0)

[error] 'typing.Any' imported but unused

(F401)